src/dispatch/static/dispatch/src/search/EditDialog.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <ValidationObserver v-slot="{ invalid, validated }">
+    <v-navigation-drawer
+      v-model="showCreateEdit"
+      app
+      clipped
+      right
+      width="500"
+      :permanent="$vuetify.breakpoint.mdAndDown"
+      class="overflow-y-auto"
+    >
+      <template #prepend>
+        <v-list-item two-line>
+          <v-list-item-content>
+            <v-list-item-title class="title"> Edit </v-list-item-title>
+            <v-list-item-subtitle>Search Filter</v-list-item-subtitle>
+          </v-list-item-content>
+          <v-btn
+            icon
+            color="info"
+            :loading="loading"
+            :disabled="invalid || !validated"
+            @click="save()"
+          >
+            <v-icon>save</v-icon>
+          </v-btn>
+          <v-btn icon color="secondary" @click="closeCreateEdit">
+            <v-icon>close</v-icon>
+          </v-btn>
+        </v-list-item>
+        <v-card flat>
+          <v-card-text>
+            <v-container grid-list-md>
+              <v-layout wrap>
+                <v-flex xs12>
+                  <ValidationProvider name="Name" rules="required" immediate>
+                    <v-text-field
+                      v-model="name"
+                      slot-scope="{ errors, valid }"
+                      :error-messages="errors"
+                      :success="valid"
+                      label="Name"
+                      hint="Name of type."
+                      clearable
+                      required
+                    />
+                  </ValidationProvider>
+                </v-flex>
+                <v-flex xs12>
+                  <ValidationProvider name="Description" rules="required" immediate>
+                    <v-textarea
+                      v-model="description"
+                      slot-scope="{ errors, valid }"
+                      :error-messages="errors"
+                      :success="valid"
+                      label="Description"
+                      hint="Description of type."
+                      clearable
+                      required
+                    />
+                  </ValidationProvider>
+                </v-flex>
+              </v-layout>
+              <div class="text-body-2 ml-1 mt-2">Individuals</div>
+              <v-divider class="mt-2" />
+              <v-list style="max-height: 500px" class="overflow-y-auto">
+                <template v-for="individual in individuals">
+                  <v-list-item :key="individual.id">
+                    <v-list-item-content>
+                      <v-list-item-title>{{ individual.name }}</v-list-item-title>
```

Comment:
Let's use components like `IncidentParticipant` instead of `individual.name` to increase clarity on which individual is subscribed. There may be multiple individuals with the same name.

Reply:
Good idea. Updated.